docs/account/emails.md

```diff
@@ -6,7 +6,7 @@ To manage the email addresses associated with your account and your email notifi
 
 ## Updating your email addresses {: id="updating"}
 
-Codacy pulls email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.
+Codacy automatically pulls to your Codacy account the email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.
```

Comment:
I believe this can be clearer if instead of *how* (pulls), we mention *what* Codacy does ("associates with" for consistency, or perhaps "links with" to avoid repeating "associate" in the sentence).

```suggestion
Codacy automatically associates with your Codacy account the email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.
```

Reply:
```suggestion
Codacy automatically links with your Codacy account the email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.
```